lib/job.js

```diff
@@ -86,6 +87,18 @@ class Job extends BaseModel {
     }
 
     /**
+     * Return PR number. Returns -1 if this is not a PR job
+     * @return {Number}     PR number
+     */
+    prNum() {
+        if (!this.isPR()) {
+            return -1;
```

Comment:
Hmmm I didn't want it to be a getter because not every job is a PR, so it's weird that every job has a get PR number. But it's fine.